sxw.c

```diff
@@ -918,24 +948,24 @@ void _print_debuginfo(void) {
   /* sum actual total transpiration */
 	ForEachTrPeriod(p)
 	{
-		//ForEachTranspLayer(t) sum += SXW.transp[Ilp(t,p)];
-		for (t = 0; t < SXW.NSoLyrs; t++)
-			sum += SXW.transpTotal[Ilp(t, p)];
+		//ForEachTranspLayer(t) sum += SXW->transp[Ilp(t,p)];
```

Comment:
This line can probably safely be removed.

Reply:
Resolved by 2a1ff37.